README.md

```diff
@@ -1,49 +1,162 @@
+### Installing Wallet SDK
 
-- **Mobile**: Users can connect to your mobile web dapp through a deeplink to the dapp browser in [Coinbase Wallet Mobile App](https://coinbase-wallet.onelink.me/q5Sx/fdb9b250).
+Install Coinbase Wallet SDK using yarn or npm.
 
-- **Desktop**: Users can connect to your desktop app with a QR code in the [Coinbase Wallet Mobile App](https://coinbase-wallet.onelink.me/q5Sx/fdb9b250) or with the [Coinbase Wallet Chrome Extension](https://coinbase-wallet.onelink.me/q5Sx/fdb9b250).
+1. Check available versions:
 
-Wallet SDK is open-source and uses minimal dependencies for maximum security and no code bloat. Simply drop a few lines of code into your dapp and Wallet SDK takes care of the rest.
+```shell
+# yarn
+yarn info @coinbase/wallet-sdk versions
 
-## Getting started
+# npm
+npm view @coinbase/wallet-sdk versions
+```
 
-This repo uses a yarn workspace. To get started, run:
+2. Install latest version:
 
 ```shell
-yarn install
+# yarn
+yarn add @coinbase/wallet-sdk
 
-# To start the development server run the following command. This starts a nextjs app on port 3001. Any changes in the SDK become available through the app.
-yarn dev
+# npm
+npm install @coinbase/wallet-sdk
+```
 
-# To interact with the SDK directly:
-yarn workspace @coinbase/wallet-sdk "<command>"
+3. Check installed version:
 
-# To lint all files
-yarn lint
+```shell
+# yarn
+yarn list @coinbase/wallet-sdk
 
-# To typecheck all files
-yarn typecheck
+# npm
+npm list @coinbase/wallet-sdk
 ```
 
-## Installing and Upgrading
+### Upgrading Wallet SDK
 
-> The installation package for **Coinbase Wallet SDK** (formerly WalletLink) is now named `@coinbase/wallet-sdk`.
+Upgrade Coinbase Wallet SDK using yarn or npm.
 
-- This [readme](/packages/wallet-sdk/README.md) contains brief instructions to get up and running.
-- Visit our [public developer docs](https://docs.cloud.coinbase.com/wallet-sdk/docs) for more detail, including samples for integrating Coinbase Wallet using libraries like [web3-react](https://github.com/Uniswap/web3-react), [web3modal](https://github.com/Web3Modal/web3modal), [Web3-Onboard](https://docs.blocknative.com/onboard), and [wagmi](https://wagmi.sh/).
+#### yarn/npm
+
+1. Compare installed version with latest:
+
+```shell
+# yarn
+yarn outdated @coinbase/wallet-sdk
+
+# npm
+npm outdated @coinbase/wallet-sdk
+```
+
+2. Update to latest:
+
+```shell
+# yarn
+yarn upgrade @coinbase/wallet-sdk --latest
+
+# npm
+npm update @coinbase/wallet-sdk
+```
+
+### Basic Usage
+
+#### 1. Initialize SDK
+
+```
+const sdk = new CoinbaseWalletSDK40({
+  appName: 'SDK Playground'
+});
+```
+
+#### 2. Make web3 Provider
+
+```
+const provider = sdk.makeWeb3Provider();
+
+```
+
+#### 3. Request accounts to initialize connection to wallet
+
+```
+const addresses = provider.request('eth_requestAccounts')
+```
+
+#### 4. Make more requests
+
+```
+provider.request('personal_sign', ['test message', addresses[0]])
```

Comment:
```suggestion
provider.request('personal_sign', [`0x${Buffer.from('test message', 'utf8').toString('hex')}`, addresses[0]])
```

Reply:
why do we need this? Kind of strange because it causes the message to be illegible in the scw popup. Is that an issue on the FE side?

Reply:
hmmm in my testing, message will be illegible if the message is not hex string tho